### PR TITLE
Expand readme.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: services/validation/README.md [hunk 1]
```diff
@@ -1,18 +1,44 @@
...
 
-#### Use this script to verify if your source files are in the correct format for [Sourcify](https://github.com/ethereum/sourcify)
+The Sourcify Validation script helps you re-compile and source-verify your Solidity smart contracts.
+
+If Solidity source files are spread across multiple files, re-compiling them is often complicated.
+To help with re-compilation, the Solidity compiler generates a file called "metadata.json" that
+contains all settings, filenames and file hashes required to re-compile the contract into the exact
+same binary.
+
+Several tools, including truffle and buidler store this file with each compilation.
```
Comment (chriseth) [inline services/validation/README.md:10]: We should provide some links here.

File: services/validation/README.md [hunk 2]
```diff
@@ -1,18 +1,44 @@
...
+that you just need to send to the compiler, to etherscan, or whatever service you use
+for verification.
+
+If you use sourcify, this step is done for you. All you need to do is upload the files
+you provided to Sourcify-Validation.
+
 
 ## Example - multiple sources
 ### Command
 `sourcify-validation Escrow.sol Main.sol Owned.sol provableAPI_0.6.sol Savings.sol metadata.json`
```
Comment (chriseth) [inline services/validation/README.md:33]: Maybe change this example to one that uses a build tool that already has the metadata somewhere?

File: services/validation/README.md [hunk 3]
```diff
@@ -22,6 +48,22 @@ Savings (browser/Savings.sol):
...
+
+`sourcify-validation -j browser/Savings.sol:Savings . | solc-linux-amd64-v0.6.11+commit.5ef660b1 --standard-json - > output.json`.
+
+If you use `jq`, it will nicely format it:
+
+### Output
+```
+sourcify-validation -j browser/Savings.sol:Savings . | solc-linux-amd64-v0.6.11+commit.5ef660b1 --standard-json - | jq
+{
+  ...
```
Comment (chriseth) [inline services/validation/README.md:62]: We should add some actual data here.

File: services/validation/README.md (hunk 2 above)
Comment (chriseth) [reply services/validation/README.md:33]: Ah, I see the example is the last - maybe we should combine them somehow?

**Comment by FabijanC** [comment]
@chriseth, your proposals came in last night about the same time as we published the then new version of README. So I copied them into the newest version of the documentation that can be found in the latest commit of this PR: https://github.com/ethereum/sourcify/pull/282

**Pull request closed without merging**